bundle-workflow/tests/tests_test_workflow/test_test_args.py

```diff
@@ -0,0 +1,188 @@
+import logging
+import os
+import unittest
+from unittest.mock import patch
+
+from test_workflow.test_args import TestArgs
+
+
+class TestTestArgs(unittest.TestCase):
+
+    ARGS_PY = os.path.realpath(
+        os.path.join(os.path.dirname(__file__), "../../src/run_bwc_test.py")
+    )
+
+    @patch(
+        "argparse._sys.argv",
+        [
+            ARGS_PY,
+            "--s3-bucket",
+            "xyz",
+            "--opensearch-version",
+            "1.1.0",
+            "--build-id",
+            "30",
+            "--architecture",
+            "x64",
+            "--test-run-id",
+            "6",
+        ],
+    )
+    def test_s3_bucket(self):
+        self.assertEqual(TestArgs().s3_bucket, "xyz")
```

Comment:
Because the `@patch` calls are all the same, combine the tests into a single one that `self.assertEqual(...)` all the things, reduce copy-paste.